samples/android-integration/permissions/src/main/java/com/badoo/ribs/samples/permissions/rib/PermissionsRib.kt

```diff
@@ -0,0 +1,9 @@
+package com.badoo.ribs.samples.permissions.rib
+
+import com.badoo.ribs.android.permissionrequester.CanProvidePermissionRequester
+import com.badoo.ribs.core.Rib
+
+interface PermissionsRib : Rib {
```

Comment:
```suggestion
interface PermissionsSample : Rib {
```

No need to mention Rib in the actual name. (Also, you can use the plugin to rename every file in a Rib automatically, no need to do one by one)